README.md

```diff
@@ -89,6 +89,7 @@ CIP Editors meetings are public, recorded, and [published on Youtube](https://ww
 | 1853 | [HD (Hierarchy for Deterministic) Stake Pool Cold Keys for Cardano](./CIP-1853/) | Active |
 | 1854 | [Multi-signatures HD Wallets](./CIP-1854/) | Draft |
 | 1855 | [Forging policy keys for HD Wallets](./CIP-1855/) | Active |
+| 1694 | [A proposal for entering the Voltaire phase](./CIP-1694) | Proposed |
```

Comment:
```suggestion
| 1694 | [A proposal for entering the Voltaire phase](./CIP-1694) | Draft |
```
Merge as Draft instead?

Reply:
We don't have a `draft` status anymore since the last CIP-0001 update. Sorry, I put `draft` in my original message but that's a brain fart. Merging it as: `proposed` it is.

Reply:
There is no longer a `Draft` status, so merge as `Proposed`.